### PR TITLE
[NET-1253] use `Date` instead of `number` for `atTime`

### PR DESCRIPTION
**Pull request by haardikk21**
# Change `atTime` argument to use `Date` instead of `number`

## Description

title

## How Has This Been Tested?

Previous tests were modified to conform

## Definition of Done

Before submitting this PR, please make sure:

- [x] The work addresses the description and outcomes in the issue
- [x] I have added relevant tests for new or updated functionality
- [x] My code follows conventions, is well commented, and easy to understand
- [x] My code builds and tests pass without any errors or warnings
- [x] I have tagged the relevant reviewers
- [ ] I have updated the READMEs of affected packages
- [ ] I have made corresponding changes to the documentation
- [ ] The changes have been communicated to interested parties

## References:

Please list relevant documentation (e.g. tech specs, articles, related work etc.) relevant to this change, and note if the documentation has been updated.

### CODE REVIEW

**Review by ukstv — APPROVED** [review]

**Review by stephhuynh18 — APPROVED** [review]

File: src/did.ts [hunk 1]
```diff
@@ -315,7 +315,8 @@ export class DID {
       signerDid === options.capability.p.aud
     ) {
       Cacao.verify(options.capability, {
-        atTime: options.atTime,
+        // CACAO requires `atTime` in seconds
```
Comment (stbrody) [inline src/did.ts:318]: Seems like we should probably be making the same change to the Cacao library?

**Review by stbrody — APPROVED** [review]
lgtm

File: src/did.ts (hunk 1 above)
Comment (haardikk21) [reply src/did.ts:318]: yeah, maybe I should block this PR until I make the same change in CACAO

Comment (haardikk21) [reply src/did.ts]: resolved. merging.

**Pull request merged**